src/jit/lowerxarch.cpp

```diff
@@ -2308,11 +2308,11 @@ void Lowering::ContainCheckSIMD(GenTreeSIMD* simdNode)
 //
 bool Lowering::IsContainableHWIntrinsicOp(GenTreeHWIntrinsic* containingNode, GenTree* node, bool* supportsRegOptional)
 {
-    NamedIntrinsic      containingIntrinsicID = containingNode->gtHWIntrinsicId;
-    HWIntrinsicCategory category              = HWIntrinsicInfo::lookupCategory(containingIntrinsicID);
+    NamedIntrinsic      containingintrinsicId = containingNode->gtHWIntrinsicId;
```

Comment:
I think you should leave the spelling as-is. "Intrinsic" should be capitalized.

Reply:
Will fix, this was an accident